unyt/mpl_interface.py

```diff
@@ -164,13 +181,19 @@ def label_style(self, label_style="()"):
         def __enter__(self):
             registry[unyt_array] = unyt_arrayConverter()
             registry[unyt_quantity] = unyt_arrayConverter()
+            self._enabled = True
 
         def __exit__(self, exc_type, exc_val, exc_tb):
             registry.pop(unyt_array)
             registry.pop(unyt_quantity)
+            self._enabled = False
 
         def enable(self):
             self.__enter__()
 
         def disable(self):
-            self.__exit__(None, None, None)
+            if self._enabled:
+                self.__exit__(None, None, None)
+
+
+__all__ = ["matplotlib_support"]
```

Comment:
Can you move this to the top of the file, below the imports? I think that’s more idiomatic and discoverable.